.github/workflows/upload_all.yml

```diff
@@ -1,26 +1,132 @@
-name: Upload to all stores
-on: [workflow_dispatch]
+name: Upload
+
+on:
+  workflow_dispatch:
+    inputs:
+      targets:
+        type: choice
+        description: Where to upload
+        options:
+        - all
+        - google-play
+        - app-store
+
 jobs:
-  call_upload_google_play:
-    name: Upload to Google Play
-    uses: ./.github/workflows/upload_google_play.yml
-    secrets:
-      POLYPOD_POLYPEDIA_REPORT_URL: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_URL }}
-      POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION: ${{ secrets.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION }}
-      POLYPOD_ANDROID_UPLOAD_KEYSTORE_FILE: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEYSTORE_FILE }}
-      POLYPOD_ANDROID_UPLOAD_KEYSTORE_PASS: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEYSTORE_PASS }}
-      POLYPOD_ANDROID_UPLOAD_KEY_ALIAS: ${{ secrets.POLYPOD_ANDROID_UPLOAD_KEY_ALIAS }}
-      POLYPOD_ANDROID_STORE_SERVICE_ACCOUNT_JSON: ${{ secrets.POLYPOD_ANDROID_STORE_SERVICE_ACCOUNT_JSON }}
+  build_test_upload:
+    name: Build, test, and upload
+    runs-on: macos-11
+    steps:
+      - name: Fail if uploading from non-release branch
+        if: |
```

Comment:
Just took this over (in spirit) from the Android workflow. I did verify that this produces a failure as expected.

I don't think the branch filtering available for e.g. pushes works for `workflow_dispatch`, does it? There the user basically selects the branch to build on, and we need to somehow restrict the list of branches they can use, this has been our best idea so far I guess? But like I said, I just kept this working the way it was.

Reply:
As long as it's available as a variable, it's happening in that branch, so you should be able to filter. Let me check anyway.

Reply:
OK, no, you can't, but there are other options
* Make the user select the branch by defining it as an input
* Filtering at the job level, instead of the step level.

This is OK anyway, if you want it that way.

Reply:
I think if we specify it as an input, the user basically has to select it **twice**. I think this approach can survive this refactoring, given how it's the best I can think of. It's not a big deal to move to a better approach later if we find one.